data/crac-creation/crac-creator-csa-profiles/src/main/java/com/powsybl/openrao/data/craccreation/creator/csaprofile/craccreator/NotEnabledException.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openrao.data.craccreation.creator.csaprofile.craccreator;
+
+import com.powsybl.openrao.data.craccreation.creator.api.ImportStatus;
+import com.powsybl.openrao.data.craccreation.util.OpenRaoImportException;
+
+/**
+ * @author Mohamed Ben-rejeb {@literal <mohamed.ben-rejeb at rte-france.com>}
+ */
+public class NotEnabledException extends OpenRaoImportException {
+
+    public NotEnabledException(ImportStatus importStatus, String msg) {
+        super(importStatus, msg);
+    }
```

Comment:
I understand the idea of this exception class, but it appears OpenRaoImportException has no child class for all the other native CRAC formats.

@phiedw @pet-mit what do you think about this? Should we keep OpenRaoImportException or can we use this new class?

Reply:
Imo, keeping NotEnabledException would require a future refactoring to create specific classes for "common" import exceptions. To discuss.

Reply:
I agree with thomas, I think for now an OpenRaoImportException is probably better. Maybe we should open an issue to one day change all the exceptions to more specific ones? (including in the optimization part, not just the import)

Reply:
I also agree with you and wasn't convinced creating a specific exception especially like Thomas said OpenRaoImportException has no child class.
but OpenRaoImportException can't solve my problem on this code design. I want to create different behaviours inside PstRangeActionCreator depending on the use case: 
Allow a group having a pst not enabeled to be imported (ignoring only that pst)
Not allow the import of any pst of a group having a single not adequate pst (for example a pst having more than ONE StaticPropertyRange with direction RelativeDirectionKind.up) .
To make that possible i had to create a specific exception but of course we can discuss other possibilities